ush/generate_FV3LAM_wflow.sh

```diff
@@ -198,12 +200,16 @@ settings="\
   'nnodes_make_ics': ${NNODES_MAKE_ICS}
   'nnodes_make_lbcs': ${NNODES_MAKE_LBCS}
   'nnodes_run_fcst': ${NNODES_RUN_FCST}
+  'nnodes_run_anal': ${NNODES_RUN_ANAL}
   'nnodes_run_post': ${NNODES_RUN_POST}
 #
 # Number of cores used for a task
 #
   'ncores_run_fcst': ${PE_MEMBER01}
   'native_run_fcst': --cpus-per-task 2 --exclusive
+  'ncores_run_anal': 240
+  'native_run_anal': --cpus-per-task 2 --exclusive
+  'partition_run_anal': vjet,kjet,xjet
```

Comment:
This should be configurable consistent with the reservation settings.

Reply:
Done.

Reply:
This didn't get updated in your recent pushes.

Reply:
Forget to delete those lines.

Reply:
removed those line

Reply:
Added "native_run_anal': --cpus-per-task 2 --exclusive" back. 
Only need to remove "'partition_run_anal': vjet,kjet,xjet"